File: sdks/thunkmetrc-kotlin/wrapper/src/main/kotlin/io/github/thunkier/thunkmetrc/wrapper/models/PlantsUpdatePlantTagRequestItem.kt
package io.github.thunkier.thunkmetrc.wrapper.models

import com.fasterxml.jackson.annotation.JsonProperty



data class PlantsUpdatePlantTagRequestItem(
    @JsonProperty("id")
    val id: Int? = null,
    @JsonProperty("label")
    val label: String? = null,
    @JsonProperty("newTag")
    val newTag: String? = null,
    @JsonProperty("replaceDate")
    val replaceDate: String? = null,
    @JsonProperty("tagId")
    val tagId: Int? = null
)

File: sdks/thunkmetrc-go/wrapper/models/plants_update_plant_tag_request_item.go
package models

type PlantsUpdatePlantTagRequestItem struct {
    Id int `json:"Id,omitempty"`
    Label string `json:"Label,omitempty"`
    NewTag string `json:"NewTag,omitempty"`
    ReplaceDate string `json:"ReplaceDate,omitempty"`
    TagId int `json:"TagId,omitempty"`
}
